stdlib/int64.gr

```diff
@@ -281,18 +520,37 @@ export let rec gte = (x: Int64, y: Int64) => {
   ret
 }
 
-// Int64 bit/logical operations
+/**
+ * @section Logical: Int64 Logical Functions.
```

Comment:
This actually needs a description. The description shouldn't define the title using the title itself. cc @ospencer